src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -76,5 +77,11 @@
     
     <bean id="arEmailService" parent="arEmailService-parentBean"
         class="edu.cornell.kfs.module.ar.service.impl.CuAREmailServiceImpl" />
-
+        
+    <bean id="cuVerifyBillingFrequencyService" parent="verifyBillingFrequencyService-parentBean"/>
```

Comment:
Just to be sure, because our custom verify billing frequency is being named in spring differently than it is in base code, our verify service will only be used in contractsGrantsInvoiceCreateDocumentService.  Is that desirable?